tiatoolbox/models/architecture/mapde.py

```diff
@@ -77,7 +77,8 @@ def __init__(
         min_distance: int = 4,
         threshold_abs: float = 250,
         num_classes: int = 1,
-    ):
+    ) -> None:
+        """Initializes :class:`MapDe`."""
```

Comment:
```suggestion
        """Initialize :class:`MapDe`."""
```

Reply:
Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/625/commits/c2e1cd4ccfd6aaeb32664f23d4f6634cddb90090